examples/react-apollo/src/profile/mfa.tsx

```diff
@@ -0,0 +1,28 @@
+import { useConfigMfa } from '@nhost/react'
+import { useState } from 'react'
+import { Button, Input, Panel } from 'rsuite'
+
+export const Mfa: React.FC = () => {
+  const [code, setCode] = useState('')
+  const { generate, activate, isActivated, isGenerated, qrCode } = useConfigMfa(code)
```

Comment:
I feel these variable names (`generate`, `activate`, `qrCode`) could be a bit more descriptive.

Generate what?
Activate what?
qrCode - is this a base64 string? URL? or something else?

Reply:
- `generate` the QR code
- `activate` MFA for the user from the code that we get after scanning the QR code
- `qrCode` is the base64 image that is generated by hasura-auth and that can be used as a `src` property in `<img/>` like `<img src={qrCode} />`

What about:
- `generate` -> `generateQrCode`
- `activate` -> `activateMfa`

Reply:
generate -> generateQrCode
activate -> activateMfa

looks good 👍 

About `qrCode`. What about something like `qrCodeDataUrl` (https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs)?

Or does the `qrCode` includes the `data:...`?

Reply:
It does - we're using the [`toDataURL`](https://github.com/soldair/node-qrcode#nodejs) function on the hasura-auth side.

Changed names in https://github.com/nhost/nhost/pull/273/commits/9b942a59bc33d30fe5fec05fd97f26d710295e7a
I also renamed to `qrCodeDataUrl`

Generally speaking, when I'm picking names, I ask myself the question "could it be something else" For instance can `qrCode` be something else than a data uri? If yes, then we need to be explicit. Can it generate something else than a qrCode, can it activate something else than MFA, etc.

Reply:
Yeah, maybe I'm picky about the names. And there is no "right" or "wrong" really. Naming and cache. Very hard.

I could argue for both your initial naming and mine.